airflow/www/static/js/cluster-activity/live-metrics/Health.tsx

```diff
@@ -81,7 +81,16 @@ const HealthSection = ({
 
 const Health = (props: CenterProps) => {
   const { data, isError } = useHealth();
-
+  let dagProcessorHealthComponent = null;
+  if (standaloneDagProcessor != undefined && standaloneDagProcessor.toString().toLowerCase() === "true") {
```

Comment:
Where is this variable coming from?

Reply:
the `standaloneDagProcessor` variable?

I defined it as a global variable here:
https://github.com/erdos2n/airflow/blob/0c8ef2e3967e3baacdf8f702d854cdf03cfdca06/airflow/www/static/js/index.d.ts#L25 

I grab the value in the `views.py` file

https://github.com/erdos2n/airflow/blob/0c8ef2e3967e3baacdf8f702d854cdf03cfdca06/airflow/www/views.py#L1054